GOVERNANCE.md

```diff
@@ -0,0 +1,71 @@
+# Appium Project Governance
+
+The Appium Project wants as much as possible to operate using procedures that
+are fair, open, inviting, and ultimately good for the community. For that
+reason we find it valuable to codify some of the ways that the Project goes
+about its day-to-day business. We want to make sure that no matter who you are,
+you have the opportunity to contribute to Appium. We want to make sure that no
+one corporation can exert undue influence on the community or hold the Project
+hostage. And likewise we want to make sure that corporations which benefit from
+Appium are also incentivized to give back.
+
+### The Technical Committee
+
+The project is officially led by a Technical Committee (TC), currently
+consisting of [@jlipps](https://github.com/jlipps),
+[@bootstraponline](https://github.com/bootstraponline),
+[@penguinho](https://github.com/penguinho), and
+[@dandoveralba](https://github.com/dandoveralba), who all represent different
+companies and different types of contribution to the Appium Project. The TC is
+responsible for high-level decisions (like establishing a feature roadmap,
+accepting sponsorship, organizing conferences, etc...), adjudicating conflicts
+between other members of the community, and making changes to the project's
+governance processes.
+
+* Membership in the TC is granted by unanimous vote of current TC members.
+* TC members can step down at any time.
+* No more than 1/4 of TC members may belong to the same company.
+* Changes to project governance require unanimous vote of the TC.
+
+### Project Committers
+
+Most of the project maintenance is handled by committers, community members who
+have been granted commit access to the Appium organization on GitHub. The
+responsibilities of committers include: reviewing and merging pull requests,
+walking through changes with developers, weighing in on architectural issues,
+and so on.
+
+* Any two TC members can decide to make someone a project committer, providing that person has shown an appropriate understanding of the Appium codebase through prior contributions of code.
+* In general any committer can review and merge a PR. In general committers should only merge code they are qualified to review, which might entail pinging another committer who has greater ownership over a specific code area.
+* Debates between committers about whether code should be merged should happen in GitHub pull requests.
+* Proposals for large changes to the project's code (architectural changes, etc...) should be brought forward as a GitHub issue, and all committers should be pinged so they can weigh in on the discussion if desired. Substantial changes, whether in proposal stage or in pull request stage should be signed off on by 1 TC member and at least 2 other committers. To assist in the discussion, a small POC can be undertaken on a subset of the Appium ecosystem and raised as a strawman PR, to give flesh to the proposal and make discussion more helpfully concrete.
```

Comment:
Should we make an issue label for these sorts of issues, for organizational sake?